src/VisualStudio/TestUtilities/VisualStudioInstanceFactory.cs

```diff
@@ -20,24 +20,30 @@ public sealed class VisualStudioInstanceFactory : IDisposable
         internal static readonly string VsCommon7Folder = Path.GetFullPath(IntegrationHelper.GetRegistryKeyValue(Registry.LocalMachine, VsRegistryRoot, "InstallDir").ToString());
 
         internal static readonly string VsExeFile = Path.Combine(VsCommon7Folder, "devenv.exe");
-        internal static readonly string VsLaunchArgs = $"{(string.IsNullOrWhiteSpace(Settings.Default.VsRootSuffix) ? "/log" : $"/rootsuffix {Settings.Default.VsRootSuffix}")} /log";
+
+        internal string VsLaunchArgs(string rootSuffix) => $"{(string.IsNullOrWhiteSpace(rootSuffix) ? "/log" : $"/rootsuffix {rootSuffix}")} /log";
```

Comment:
Please change back. The app.config file defines the `RootSuffix` setting, which can be changed in the TestImpact repo when the tests are run.